ccd-definition/CaseEvent/User/UserEvents-SDO-nonprod.json

```diff
@@ -15,5 +15,22 @@
     "EndButtonLabel": "Submit",
     "RetriesTimeoutAboutToStartEvent": 0,
     "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "NotSuitable_SDO",
+    "Name": "Not suitable for SDO",
+    "Description": "Not suitable for SDO",
+    "DisplayOrder": 18,
+    "PreConditionState(s)": "JUDICIAL_REFERRAL",
+    "PostConditionState": "PROCEED_TO_HERITAGE_SYSTEM",
```

Comment:
4106 said that if the process is initiated by a legal-advisor the state does not change, only if it is initiated by a judge profile. That's part of the Camunda process there